CourseGrab/ViewControllers/Home/HomeTableViewCell.swift

```diff
@@ -161,6 +184,15 @@ class HomeTableViewCell: UITableViewCell {
             }
         }
     }
+    func setBackgroundColor(isHighlighted: Bool){
+        if isHighlighted{
+            self.containerView.backgroundColor = UIColor(red: 220/255, green: 220/255, blue: 220/255, alpha: 1)
+        }
+        else{
```

Comment:
Lots of spacing issues here

```suggestion
    func setBackgroundColor(isHighlighted: Bool) {
        if isHighlighted {
            containerView.backgroundColor = UIColor(red: 220 / 255, green: 220 / 255, blue: 220 / 255, alpha: 1)
        } else {
```

And anyway, I suggest this more idiomatic approach:

```
    var isHighlighted: Bool {
        didSet {
            let highlightColor = UIColor(red: 220/255, green: 220/255, blue: 220/255, alpha: 1)
            containerView.backgroundColor = isHighlighted ? highlightColor : .white
        }
```